app/src/androidTest/java/ch/epfl/sdp/musiconnect/events/MyEventPageTests.java

```diff
@@ -133,4 +134,21 @@ public void testDeleteYesButtonClick() {
 
         assertTrue(eventPageRule.getActivity().isFinishing());
     }
+
+    @Test
+    public void testMapButtonGoesToMap(){
+        Event event = md.getDummyEvent(4);
+        Intent intent = new Intent();
+        intent.putExtra("eid", event.getEid());
+        eventPageRule.launchActivity(intent);
+
+        waitSeconds(3);
+
+        onView(withId(R.id.toMap)).perform(click());
+        onView(withId(android.R.id.button1)).perform(click());
+
+        waitSeconds(3);
+
+        intended(hasComponent(MapsActivity.class.getName()));

```

Comment:
Good test

Reply:
An event can't be created without a location anymore, can it?

Reply:
Yes the geocoder can fail and return null, or we can set a location as « at the studio »

Reply:
I though if the geocoder failed, it wouldn't upload the event; doesn't make much sense to have an event without a valid location